allensdk/brain_observatory/filtered_running_speed/_schemas.py

```diff
@@ -0,0 +1,60 @@
+import argschema
+from argschema.fields import Nested
+
+
+class MultiStimulusRunningSpeedInputParameters(argschema.ArgSchema):
+    output_path = argschema.fields.InputFile(
```

Comment:
I think you meant `argschema.fields.OutputFile` (you put `InputFile`)

Reply:
I've updated this.